coil-base/src/main/java/coil/util/OkHttpClients.kt

```diff
@@ -0,0 +1,28 @@
+package coil.util
+
+import android.content.Context
+import okhttp3.Cache
+import okhttp3.Dispatcher
+import okhttp3.OkHttpClient
+
+/**
+ * [OkHttpClient] utilities for Coil.
+ */
+object OkHttpClients {
```

Comment:
Can you remove this + make `applyCoilOptimizations` a top level function?

Reply:
Done in 9384a04